Firestore/core/src/remote/grpc_connection.cc

```diff
@@ -151,7 +153,7 @@ std::unique_ptr<grpc::ClientContext> GrpcConnection::CreateContext(
 
   auto context = absl::make_unique<grpc::ClientContext>();
   if (token.data()) {
-    context->set_credentials(grpc::AccessTokenCredentials(MakeString(token)));
```

Comment:
Is it safe to not call `set_credentials`; in other words, is it guaranteed that the only thing `set_credentials` does is to add the header? Otherwise, perhaps the header should be added _in addition_ to calling `set_credentials`, not instead of it?

Reply:
It's safe to not call `set_credentials`. The only observable effect it has on the HTTP messages that are sent is to add the `authorization` header.